ukrdict/ukrdict/ukrdict.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+
+import requests
+from bs4 import BeautifulSoup
+
+
+class WordNotFound(Exception):
```

Comment:
```
class WordNotFound(Exception):
    """Exception class for ukrdict project."""
```
We can replace this. Without `pass`.
P.S. Maybe rewrite documentation, because I'm not strong in writing stronger text.

Reply:
> Also, I prefer `black` and `isort` for python projects, can you please to do
Done.

Also, changed exception class.